.travis.yml

```diff
@@ -25,6 +25,10 @@ install:
 jobs:
   fast_finish: true
   include:
+    - language: shell
+      name: checking syntax of shell script
+      script:
+        - bash -c 'shopt -s globstar nullglob; shellcheck ./.ci/*.sh' || true
```

Comment:
Why `|| true` is required?
If it failing let's fail.

Reply:
Why bash -c is required ?

Reply:
I was using an outdated command I guess. I have changed it.